app/views/candidate_mailer/reinstated_offer.text.erb

```diff
@@ -4,8 +4,11 @@ Dear <%= @application_form.first_name %>
 
 The course starts in <%= @course_option.course.start_date.to_fs(:month_and_year) %>.
 
-<% if @conditions.any? %>
+
+<% if @application_choice.offer.conditions.any? { |condition| condition.status != 'met'} %>
```

Comment:
Instead of looping in Ruby can we use SQL/active record to do this?
Example: `unless @application_choice.offer.conditions.where(status: 'met').count.zero?`
Or use a scope: `unless @application_choice.offer.conditions.met_count.zero?` and then add a scope/method to the condition model?

Reply:
Made this change, what benefit does this give us? Speed?